### DISCUSSION TITLE
ROG Zephyrus G GA502DU backlight not turning on on startup

### DISCUSSION THREAD

**Discussion by pko711** [seerge/g-helper#1070] [General]
G-helper on ROG Zephyrus G GA502DU does not turn on my laptops backlight on startup. However, I can manually turn it on and off.

**Comment by seerge** [comment]
@pko711 hello, check your logs :) app initiates keyboard and sets brightness on each startup

**Reply by pko711** [reply]
It's odd sometimes the backlight turns on at startup and other times it doesn't. Here are the logs: 
[log.txt](https://github.com/seerge/g-helper/files/12329610/log.txt)

**Comment by seerge** [comment]
Does this build make a difference ?
[GHelper.zip](https://github.com/seerge/g-helper/files/12329670/GHelper.zip)

**Reply by pko711** [reply]
Nope, the same problem is happening.

**Reply by seerge** [reply]
Well, app clearly sends all commands every start (mode / and brightness as well).

You can try to disable keyboard mode setting at all, mb that will affect you

1. Quit app
2. Add following line to ``%appdata%\ghelper\config.json`` and save
```
"skip_aura" : 1,
````
3. Run app again :)

**Reply by pko711** [reply]
Even after doing that, sometimes on startup the keyboard backlight does not turn on. I can see the log telling the keyboard to turn the backlight on but it is not turning on. The application is running on startup.

**Reply by seerge** [reply]
Well, i can’t really help here :)

**Reply by pko711** [reply]
I appreciate you trying to help, thank you!

**Reply by seerge** [reply]
That’s strange that keyboard is generally off on boot, as on modern models it just remembers state it had before

**Reply by pko711** [reply]
Yeah, I have no clue why its like that.

**Reply by pko711** [reply]
After testing it a bit more it seems that it is now working again. It's very weird that it wasn't working before and now all of a sudden it works. Thank you for your help!
